.eslintrc.yaml

```diff
@@ -23,7 +23,7 @@ rules:
   # if they are unused.
   "@typescript-eslint/no-unused-vars": ["error", { "args": "none" }]
   # For overloads.
-  no-dupe-class-members: off
+  no-dupe-class-members: 0
```

Comment:
0 === off (my yaml linter complained about this so I fixed it)

Reply:
Um, it complained about this `off` but not all the other `off`s below it, confusingly?
I think `off` makes sense here

Reply:
fixed!